cmd/bad-key-revoker/main.go

```diff
@@ -76,8 +78,11 @@ func (bkr *badKeyRevoker) countUncheckedKeys(ctx context.Context) (int, error) {
 		&count,
 		`SELECT COUNT(*)
 		FROM (SELECT 1 FROM blockedKeys
-		WHERE extantCertificatesChecked = false
+		WHERE extantCertificatesChecked = 0
+		OR (extantCertificatesChecked < 2 AND added BETWEEN NOW() - INTERVAL ? SECOND AND NOW() - INTERVAL ? SECOND)
```

Comment:
We typically stay away from using `NOW()` directly, instead we would pass the time from the `badKeyRevoker.clk`.